state/watcher/txnwatcher.go

```diff
@@ -41,7 +38,7 @@ const (
 
 	txnWatcherShortWait = 10 * time.Millisecond
 
-	maxSyncRetries           = 8
+	maxResumeRetries         = 1
```

Comment:
Do we need to recruit back-off strategies if we're only allowing a single retry?

Reply:
Yeah I think this is the part I want to remove, the mongo specifications from memory are:
- `getMore` on a cs is always retryable once
- `getMore` on a cs is retryable if the error code isn't one of a well defined set of errors.
- `getMore` is not retryable on error 43 (cursor not found)
- if `getMore` returns 43, then we can resume by making a second call to `aggregate` with the resumption token
- if `aggregate` errors, then all errors are failures, this is where we exit out.

I'd like to adjust the backoff to deal with `getMore` retries, not `aggregate` retries, which is always only once on certain errors from a `getMore`. So still some work to do here.